src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine.kt

```diff
@@ -79,13 +79,13 @@ import kotlin.time.Duration.Companion.days
 
 @Suppress("UNCHECKED_CAST")
 @JsExport
-// @Serializable
 open class TradingStateMachine(
     private val environment: V4Environment?,
     private val localizer: LocalizerProtocol?,
     private val formatter: Formatter?,
     private val maxSubaccountNumber: Int,
     private val useParentSubaccount: Boolean,
+    val staticTyping: Boolean = false,
```

Comment:
can be private i think?

Reply:
There are extension functions reading it.